templates/GenespotRE/user_detail_saml.html

```diff
@@ -84,6 +84,7 @@ <h4 class="modal-title" id="unlink-nih-account-header">Unlink {{ user_details.em
 
     <script type="text/javascript"> 
         var expiry_time = new Date("{{ user_details.NIH_assertion_expiration|safe }}".replace(" ","T"));
-        document.getElementById('NIH_expiration').innerHTML = expiry_time; 
+        if(expiry_time != null)
+            document.getElementById('NIH_expiration').innerHTML = expiry_time; 
     </script>
```

Comment:
So the error is that document.getElementById('NIH_expiration').innerHTML is null, and that's null because on this page document.getElementById('NIH_expiration').innerHTML only loads when there's an NIH account to bind to. So the fix would be to bind document.getElementById('NIH_expiration').innerHTML to a variable and test for null on that.

The expiry_time may also be null, because there will be no NIH_assertion_expiration if there's no NIH account, but the real error IIRC comes from the absence of the document element.